balancer-labs/frontend-v2
# Show tokens as icons

# Description

- Show popular Tokens and Tokens in Wallet as icon buttons on Invest page
- If BalAsset should be clickable, set it to be a `button` element
- Move some style overrides from `BalAssetSet` to `BalAsset`
- Add ability to pass down props to BalAsset via BalAssetSet using `balAssetProps` prop

## Type of change


## How should this be tested?
Go to Invest page and select some tokens. See that pools in the pools table are filtered by your selection.

## Visual context

| Tokens when Wallet is Connected |  Tokens when Wallet is not Connected |
|----|----|
|  ![Screen Shot 2022-06-13 at 13 13 05](https://user-images.githubusercontent.com/28311328/173389777-8d0facf1-0239-47e6-aeb5-315fe627c83b.png) |   ![Screen Shot 2022-06-13 at 13 13 35](https://user-images.githubusercontent.com/28311328/173389713-97df466f-bc8b-4a5c-8be6-18cbfba5cb4f.png)  | 

## Checklist:
